crates/aptos-telemetry-service/src/log_ingest.rs

```diff
@@ -98,7 +98,7 @@ pub async fn handle_log_ingest(
     debug!("ingesting to humio: {:?}", unstructured_log);
 
     let res = context
-        .humio_client
+        .humio_client()
         .ingest_unstructured_log(unstructured_log)
```

Comment:
qq, do we track the performance for this?

Reply:
@larry-aptos  Kind of, yes. Cloud run tracks the latency of each request per endpoint. We can technically extrapolate humio ingest latency from that, because pretty much all we do here is call the humio client.